cmd/harborAPI.go

```diff
@@ -4,29 +4,121 @@ import (
 	"bufio"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"net/http"
+	"os"
 	"strings"
 
 	"github.com/jtacoma/uritemplates"
+	"github.com/mitchellh/go-homedir"
 	"github.com/parnurzeal/gorequest"
 )
 
-var shipItURI = "http://shipit.services.dmtio.net"
-var triggerURI = "http://harbor-trigger.services.dmtio.net"
-var authAPI = "http://auth.services.dmtio.net"
-var helmitURI = "http://helmit.services.dmtio.net"
-var harborURI = "http://harbor.services.dmtio.net"
-var catalogitURI = "http://catalogit.services.dmtio.net"
-var customsURI = "https://customs.services.dmtio.net"
+const baseURL string = ".services.ec2.dmtio.net"
+
+// Config is the config for all communications in harbor
+type Config struct {
+	ShipitURI    string `json:"shipit"`
+	CatalogitURI string `json:"catalogit"`
+	TriggerURI   string `json:"trigger"`
+	AuthURI      string `json:"authn"`
+	HelmitURI    string `json:"helmit"`
+	HarborURI    string `json:"harbor"`
+	CustomsURI   string `json:"customs"`
+}
+
+func readConfig() (*Config, error) {
```

Comment:
The config management stuff feels maybe a bit much to have in this API client file.  Should we consider moving it to a different file, something like `config.go`?  That might also make more sense should we add non-api related config things in the future.